File: patch_scraper.py
import re
with open('backend/services/matchScraper.ts', 'r') as f:
    content = f.read()

fixed = content.replace('timeout: config.REQUEST_TIMEOUT', 'timeout: config.REQUEST_TIMEOUT, responseType: "text", transformResponse: [(data) => data]')

with open('backend/services/matchScraper.ts', 'w') as f:
    f.write(fixed)

File: patch_scraper.js
const fs = require('fs');
const path = 'backend/services/matchScraper.ts';
let code = fs.readFileSync(path, 'utf8');
code = code.replace(
  'timeout: config.REQUEST_TIMEOUT',
  'timeout: config.REQUEST_TIMEOUT,\n          responseType: "text",\n          transformResponse: [(data) => data]'
);
fs.writeFileSync(path, code);
